lib/rubocop/cop/rspec/factory_bot/create_list.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Prefer using create_list over n.times { create :obj } calls.
+        #
+        # @example
+        #   # bad
+        #   3.times { create :user }
+        #
+        #   # good
+        #   create_list :user, 3
+        #
+        #   # good
+        #   3.times { |n| create :user, created_at: n.months.ago }
+        class CreateList < Cop
+          MSG = 'Prefer create_list.'.freeze
+
+          def_node_matcher :n_times, '(send (int $_) :times)'
+
+          def_node_matcher :times_block_without_args?, <<-PATTERN
+            (block
+              #n_times
+              (args)
```

Comment:
Help me: does `args` have a special meaning here?

Reply:
means no arguments passed to the blog (otherwise (args will be followed by the args yielded)

Reply:
I’m asking because I don’t see it documented in https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/node_pattern.rb#L5-L91

Reply:
I'm just using `ruby-parse` to generate the ASTs. `(args) is used in some other cops too

Reply:
No longer needed, as now n.times with arguments is also flagged